frontend/src/containers/WelcomePage/WelcomePage.tsx

```diff
@@ -14,6 +15,11 @@ const WelcomePage: React.FC = () => {
       <Box
         sx={containerStyle}
       >
+         <Box
```

Comment:
You can put this Box wrapper inside the BannerButton component so you only need to call `<BannerButton />` in the landing, sign in and sign up pages